### PR TITLE
Do not set "Content-Type: text/plain" on binary requests

### PR DESCRIPTION
**Pull request by aparamon**
Currently sets `Content-Type: application/octet-stream` instead, but probably should default to `None` always.

### CODE REVIEW

**Comment by seanbrant** [comment]
Thanks! I’ll try and get this and some of the other PRs merged and released soon.

**Comment by aparamon** [comment]
@seanbrant  That would be really awesome!!
What about always defaulting to `None` Content-Type in this PR? It seems that current behavior setting `text/plain` is too smart, and different from what e.g. Werkzeug does.

CC @MuchkoM

File: wsgiadapter.py [hunk 1]
```diff
@@ -92,17 +92,22 @@ def send(self, request, *args, **kwargs):
...
+            # TODO: remove?
+            content_type = 'application/octet-stream'
         else:
             data = request.body.encode('utf-8')
+            # TODO: remove?
+            content_type = 'text/plain'
 
         environ = {
-            'CONTENT_TYPE': request.headers.get('Content-Type', 'text/plain'),
+            'CONTENT_TYPE': request.headers.get('Content-Type', content_type),
```
Comment (seanbrant) [inline wsgiadapter.py]: Thoughts on defaulting to `""` (empty string). I'm also leaning towards not automatically setting it based on the body if it's not in the request headers.

**Comment by aparamon** [comment]
@seanbrant Could you please take another look?

**Pull request merged**